### ISSUE TITLE
license on font file?

### ISSUE THREAD

**Issue by minektur** [cstrap/monaco-font#10]
What is the license on this font?

**Comment by cstrap** [comment]
I'm not the creator of the font, I found it in a website (see README.md) and the licence was not specified.  
In fact, I wrote the bash script in order to get an easy installation on a linux box and I assumed that everybody can use it for free.
In addition, for each branch (monaco-font-gophers, monaco-font-ide and monaco-font-powerline), each font inherits the licence of the original project, if specified; you can find the links on each commit messages.
After a rapid search on google, each version of the font has no declarative licence specified, there are a pletora of monaco.ttf files free for download and use.
In conclusion, I presume that the licence is unknown and it can be used for free.

**Comment by kyrias** [comment]
> In conclusion, I presume that the licence is unknown and it can be used for free.

That's a silly conclusion that completely ignores how copyright law works.

Monaco is not a freely licensed font, meaning that distributing it or any modified versions, or even just downloading it from unlicensed sources is copyright infringement, and you could be sued by Apple, the copyright holder.

**Comment by cstrap** [comment]
> That's a silly conclusion that completely ignores how copyright law works.

You're right, I'm not a lawyer and you could help me to understand how copyright works.
In fact...

> Monaco is not a freely licensed font, meaning that distributing it or any modified versions, or even just downloading it from unlicensed sources is copyright infringement, and you could be sued by Apple, the copyright holder.

Good point @kyrias did you find the licence? Could you help us, please?
The monaco font cames form here: http://www.gringod.com/2006/11/01/new-version-of-monaco-font/ and other changes came from the respective citations (gopher, powerline etc).
Again, I didn't find the license of those fonts. 

Hope you help us. Thanks.

**Comment by kyrias** [comment]
There's no actual license for it, because it's not meant to be used by others freely. If you want to use materials copyrighted by Apple, you're going to have to [ask them](https://www.apple.com/legal/intellectual-property/rightsandpermissions.html) yourself.

**Comment by cstrap** [comment]
@kyrias I understood. 
Since I don't know the origin of the fonts, I'm planning to remove all the copies from my repo and put a disclaimer that explains why.
The how to install the font on linux distros will remain.

Thanks for your help.

**Comment by cstrap** [comment]
Now the `monaco-font` repo contains only the scripts that, given an `url`, permit to download and install whatever type of fonts.

The users that install whatever type of font have to know that every fonts have a license and he/she should keep in mind that they may not be used for free.
It's better to know the license before download and install the font.

Other info: please see README.
